TYPO3.Flow/Classes/TYPO3/Flow/Security/GlobalObjectStateInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Created by IntelliJ IDEA.
+ * User: sebastian
+ * Date: 17.03.17
+ * Time: 09:58
+ */
+
+namespace TYPO3\Flow\Security;
+
+interface GlobalObjectStateInterface
```

Comment:
see https://github.com/neos/flow-development-collection/pull/893/commits/87c341f54624844abc7fee9ee73e01231db8cdb5#diff-51acb520957e5e74e17bfd768b12d9d2R875 it is important, you want to have your own global object to have the ability to add a context hash ... very important, i think @skurfuerst can easily demonstrate that with the example project given 😄